ios/chargebee_flutter.podspec

```diff
@@ -14,7 +14,7 @@ A new Flutter plugin.
   s.dependency 'Flutter'
   s.platform = :ios, '12.0'
 
-  s.dependency 'Chargebee', '1.0.25'
+  s.dependency 'Chargebee', '1.0.26'
```

Comment:
Can we start to follow semantic versioning here? Since we are releasing a new feature, lets bump it to `1.1.0`?

Reply:
nvm, missed to see that its the dependent version.